### ISSUE TITLE
Videos with format FLV don't work

### ISSUE THREAD

**Issue by leadvic** [docmost/docmost#414]
Hi there,
I tried uploading some videos into my instance of Docmost v0.4.1.

When I upload a .mp4 file it works just fine, but when I upload a .flv video I get the following:
![image](https://github.com/user-attachments/assets/c176f2fb-d224-4b64-a853-c082c63b0e73)
Is it possible to get more video formats compatible with Docmost, or if it is not, could there be a documented list of all the available video formats?

**Comment by Servostar** [comment]
Docmost makes use of the builtin `HTMLVideoElement` for displaying videos. Container and therefor codec support depends on the browser/platform (see 1). Well supported container for video formats are documented here (2).
Since the retirement of adobe flash (since Chromium version 88 (see 3)), flash video files (`.flv`) are generally not well supported especially on web browsers. I don't think there is much to be done from docmost's site here.

TLDR; In other words its a lack of browser support for a format used by a platform no longer included into recent browsers. I recommend converting the `.flv` to a more common format such as `.mp4`.

1. https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement
2. https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers
3. https://www.chromium.org/flash-roadmap/#flash-support-removed-from-chromium-target-chrome-88-jan-2021

**Comment by leadvic** [comment]
That's totally reasonable for me, thanks for that! 👍👍
